src/fst.jl

```diff
@@ -1158,3 +1158,15 @@ function eq_to_in_normalization!(fst::FST, always_for_in::Bool, for_in_replaceme
 end
 
 eq_to_in_normalization!(::FST, ::Nothing, ::String) = nothing
+
+# Check if the caller of a call is in `list`
+# Note that this also works for CSTParser.EXPR
+function caller_in_list(fst, list)
+    if is_leaf(fst[1]) && fst[1].val in list
+        return true
+    elseif !is_leaf(fst[1]) && is_leaf(fst[1][1]) && fst[1][1].val in list
+        return true
+    end
+
+    return false
+end
```

Comment:
```suggestion
function caller_in_list(fst, list::AbstractArray{String})
    if is_leaf(fst[1]) && fst[1].val::String in list
        return true
    elseif !is_leaf(fst[1]) && is_leaf(fst[1][1]) && fst[1][1].val::String in list
        return true
    end

    return false
end
```
`list` might contain `missing` (at least that's what the compiler thinks). Explicitly requiring a vector of a non-`Missing` type solves this warning. Same for the other two annotations telling the compiler that this must not be `missing`. However, now the user can't pass a tuple or other iterable to `variable_dict_indent`. Is this the correct fix for this warning? Should we allow `NTuple{N, String} where N` here as well?